scripts/us_bls/jolts/bls_jolts.py

```diff
@@ -0,0 +1,346 @@
+  total_other_seps = web_to_pandas("https://download.bls.gov/pub/time.series/jt/jt.data.7.OtherSeparations", sep="\t", expected_cols=expected_job_columns)
+
+  # Additional information about each dataframe
+  # Tuple Format: Statistical Variable name, Stat Var population, Stat Var Job Change Type If Relevant, Dataframe for Stat Var 
+  schema_mapping = [
+    ("NumJobOpening", "schema:JobPosting", "", job_openings),
+    ("NumJobHire", "dcs:BLSWorker", "Hire", job_hires),
+    ("NumSeparation", "dcs:BLSWorker", "Separation", total_seps),
+    ("NumVoluntarySeparation", "dcs:BLSWorker", "VoluntarySeparation", total_quits),
+    ("NumInvoluntarySeparation", "dcs:BLSWorker", "InvoluntarySeparation", total_layoffs),
+    ("NumOtherSeparation", "dcs:BLSWorker", "OtherSeparation", total_other_seps),
+  ]
+
+  # Combine datasets into a single dataframe including origin of data
+  jolts_df = pd.DataFrame()
+  columns_to_keep = ['series_id', 'year', 'period', 'value']
+
+  for schema_name, population_type, job_change_event, df in schema_mapping:
+    df = df.loc[:, columns_to_keep]
+    df['statistical_variable'] = schema_name
+    df['job_change_event'] = job_change_event
+    df['population_type'] = population_type
+    jolts_df = jolts_df.append(df)
+
+  # Drop non-monthly data
+  jolts_df = jolts_df.drop(jolts_df.query("period == 'M13'").index)
+
+  # Change date to ISO Format (YYYY-MM)
+  def period_year_to_iso_8601(row):
+    month = row['period'].lstrip("M")
+    year = row['year']
+    return f"{year}-{month}"
+
+  jolts_df.loc[:, 'Date'] = jolts_df.apply(period_year_to_iso_8601, axis=1)
+
+  # Add relevant columns from series information
+  jolts_df = jolts_df.merge(series_desc[['industry_code', 'region_code', 'seasonal', 'ratelevel_code']], left_on=["series_id"], right_index=True)
+
+  # Drop rate data
+  jolts_df = jolts_df.query("ratelevel_code == 'L'")
+
+  # Drop non-national data
+  jolts_df = jolts_df.query("region_code == '00'")
+
+  # Map industries
+  def jolts_code_map(row):
+    """ Maps industry code used by BLS Jolts to NAICS or BLS aggregation """
+    industry_code = row['industry_code']
+    assert industry_code in CODE_MAPPINGS
+
+    mapped_code = CODE_MAPPINGS[industry_code]
+
+    # New Jolts code
+    if ":" in mapped_code:
+      JOLTS_CODE = mapped_code.split(":")[0]
+      row['industry_code'] = "JOLTS_" + JOLTS_CODE
+
+    # NAICS
+    else:
+      row['industry_code'] = mapped_code
+    
+    return row
+
+  jolts_df = jolts_df.apply(jolts_code_map, axis=1)
+
+  def map_row_to_statistical_variable(row):
+    """ Maps a row of the dataframe to the Statistical Variable that describes it """
+    base_stat_var = row['statistical_variable']
+    industry_code = row['industry_code']
+
+    return f"dcs:{base_stat_var}_NAICS_{industry_code}"
+
+  # Build map to Statistical Variable
+  jolts_df['SeasonalAdjustment'] = jolts_df['seasonal'].apply(lambda adjustment: "Adjusted" if adjustment == "S" else "Unadjusted")
```

Comment:
I noticed your MCF had "Unadjusted" and "Adjusted".

1. These should be enum references.
2. Let's reuse https://browser.datacommons.org/kg?dcid=BLSSeasonallyUnadjusted and https://browser.datacommons.org/kg?dcid=BLSSeasonallyAdjusted

Reply:
Added.